src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -366,12 +374,23 @@ function genFragment(
   }
 
   var entityId: ?string = null;
-  var href: ?string = null;
 
   while (child) {
     if (nodeName === 'a' && child.href && hasValidLinkText(child)) {
-      href = new URI(child.href).toString();
-      entityId = DraftEntity.create('LINK', 'MUTABLE', {url: href});
+      // had to cast to Object due to https://github.com/facebook/flow/issues/1730
+      const anchor: Object = child;
+
+      let entityConfig = {};
```

Comment:
But `entityConfig` isn't reassigned anywhere, right? You just add fields to the object?